pkgs/build-support/compress-drv/default.nix

```diff
@@ -51,7 +51,11 @@
   :::
 */
 drv:
-{ formats, compressors }:
+{
+  formats,
+  compressors,
+  extraArgs ? "",
```

Comment:
Presence of `extraArgs` means we will never be able to replace `find` with something else, as this broadens up the interface up to the point where we cannot reason how it works.

AFAIK from your comment earlier, would you like to exclude some extensions? If yes, how about `excludePatterns` or so?

Reply:
I want to exclude some paths but good point.

Reply:
that's what I am using right now

```
package = pkgs.compressDrvWeb pkgs.nextcloud29 {
  excludePatterns = [ ".*(\/apps\/.*\/l10n\/).*" ];
};
```